day_two_guide/topics/increasing_docker_storage.adoc

```diff
@@ -127,6 +141,43 @@ ose-app-node01.example.com   Ready                      24m       v1.6.1+5115d70
 ose-app-node02.example.com   Ready                      24m       v1.6.1+5115d708d7
 ----
 
+*Increasing storage with a new disk*
+
+. Evacuate the node xref:evacuating-a-node[following the steps above].
+
+. Stop the `docker` and `atomic-openshift-node` services:
++
+----
+# systemctl stop docker atomic-openshift-node
+----
+
+. Resize the disk.
+
+. Reattach the disk.
+
+. Run the `docker-storage-setup` command to extend the volume groups and logical
+volumes associated with container storage:
++
+----
+# docker-storage-setup
+INFO: Volume group backing root filesystem could not be determined
+INFO: Device /dev/xvdb is already partitioned and is part of volume group docker_vol
+INFO: Device node /dev/xvdd1 exists.
+  Physical volume "/dev/xvdd1" successfully created.
+  Volume group "docker_vol" successfully extended
+----
+
+. Reconfigure `docker-storage-setup`.
```

Comment:
This should be done before running the command.

Reply:
@e-minguez Sounds like (and I could be wrong) that the above step is doing this. I've deleted the step here. Please let me know if there's something different I should do.